### DISCUSSION TITLE
Is there a way to donate to the project?

### DISCUSSION THREAD

**Discussion by Nebucatnetzer** [python-gitlab/python-gitlab#2593] [Q&A]
As the title mentions I havent' found a way to donate to this project or are you financially covered?
Which would be really good to hear :).

**Comment by nejch** [comment]
Hey @Nebucatnetzer, thanks a lot!

I'm able to work on this project as part of my job. @JohnVillalovos's work is fully voluntary though AFAIK, and maybe he could set up GitHub sponsors for his personal GitHub account but I also am not 100% of how this would be set up at the moment. Would need to explore the options here a bit. :)

**Comment by Nebucatnetzer** [comment]
No problem :).
I plan to use `python-gitlab` at my job and we have a pool from which I can donate to projects we use.
We're currently in the donating round for this year but if don't have something right at hand I can still add you next year.
For projects we use regularly I would like to do yearly donations.
It won't be a huge amount btw.
